src/core_plugins/metrics/common/metric_types.js

```diff
@@ -0,0 +1,8 @@
+export const metricTypes = [
+  'markdown',
+  'gauge',
+  'table',
+  'metric',
+  'top_n'
+];
+
```

Comment:
As mentioned below, a predicate like `isMetric(panelType: PanelType): boolean` could go well with this and make conditionals more readable.